src/Model/House.java

```diff
@@ -44,8 +44,8 @@ public House(ArrayList<Room> r, String l) {
      * @param srcJSON the file in which the house layout is contained
      * @throws JSONException if the file is not found or is incorrect
      */
-    public House(File srcJSON) throws JSONException, IOException {
-        this.rooms = Room.roomFromJSON(srcJSON);
+    public House(File srcJSON, float temperature) throws JSONException, IOException {
+        this.rooms = Room.roomFromJSON(srcJSON, temperature);
```

Comment:
This is a bit nitpicky but shouldn't the roomFromJSON function only read from JSON?
You could just run through all the rooms and set their temperature on the next line, instead of passing it on to the JSON function.

Reply:
Yeah this is really crappy code tbh.  This was the fastest and easiest way I thought of to pass the default outside temperature from Simulation to Room. I could just use a public static arg though tbh.  Will refactor.

Reply:
resolved by https://github.com/DGovi/soen343house/pull/131/commits/1e8a428642abc3eebd54d88f4c1b62b73f66d195